index.html

```diff
@@ -92,8 +92,23 @@ <h2 class="service__header">Proudly Serving Couples For 30 Years</h2>
                 <p class="service__info-text">The Wedding Shop is a multi-award-winning gift list service that has been supporting couples for three decades. It’s often a whirlwind of wedmin and emotions, but we’re here to help, and thrilled to be a part of your exciting journey.</p>
                 <button class="button service__info-btn">Our Service Explained</button>
             </div>
+        </section>
 
+        <section class="floating">
+            <div class="floating__block">
+               <h2 class="floating__title">30 Years Experience</h2>
+               <img class="floating__img" src="img/float.jpg"/>
+            </div>
+            
+            <div class="textbox">
+                <h4 class="textbox__brand-name">The Wedding Shop</h4>

```

Comment:
just want to make sure - are those h4/h3 tags coming from Figma? Or was it your call?

Reply:
Just my call, should I change them?

Reply:
They are just text on figma. 
![image](https://user-images.githubusercontent.com/113596529/217497467-50af1968-7038-4d2b-b2a5-bdb677d17c2d.png)

Reply:
On live the 'The Wedding Shop' is a p tag and 'Our Promise' is h2

Reply:
in real case scenario that would be something to be discussed with someone from SEO team, now you can just replicate our live version

Reply:
Thank you, done.